src/shared/config/config.loader.spec.ts

```diff
@@ -105,13 +109,18 @@ describe('configloader', () => {
             });
 
             it('should return validated JsonConfig', () => {
+                jest.spyOn(fs, 'existsSync').mockReturnValueOnce(true);
                 const validatedConfig: JsonConfig = loadConfigFiles();
                 expect(validatedConfig).toBeInstanceOf(JsonConfig);
                 expect(readFileSyncSpy).toHaveBeenCalledTimes(2);
             });
         });
 
         describe('When there is no secrets config', () => {
+            beforeAll(() => {
+                jest.clearAllMocks();
+            });
```

Comment:
Was this change needed? I don't think any changes you did should affect these tests